cmd/backup/script.go

```diff
@@ -159,6 +164,45 @@ func newScript() (*script, error) {
 		}
 	}
 
+	if s.c.SSHHostName != "" {
+		var authMethods []ssh.AuthMethod
+
+		if s.c.SSHPassword != "" {
+			authMethods = append(authMethods, ssh.Password(s.c.SSHPassword))
+		}
+
+		if _, err := os.Stat(s.c.SSHIdentityFile); err == nil {
+			key, err := ioutil.ReadFile(s.c.SSHIdentityFile)
+			if err == nil {
```

Comment:
I wonder if it's ok to just continue in case there is an error here? If we reach this point and `err` is not `nil` it means an identity file exists, but there was an error reading it (e.g. permissions or whatever). I would expect the script to fail in such cases instead of trying to continue without the key file. Any reason for this behavior I am missing?